schema/testimonials/2.0/schema.json

```diff
@@ -1,601 +1,211 @@
 {
-  "$id": "opencerts/v2.1",
+  "$id": "https://schemas.accredify.io/opencerts/testimonials/2.0/schema.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "definitions": {
-    "identityProofDns": {
+    "Testimonial": {
```

Comment:
Is this meant to Pascal Case or Camel Case?

Reply:
Camel case would be better. Was following their previous implementation